_includes/geek-details/routing-calico.md

```diff
@@ -0,0 +1 @@
+{{site.prodname}} distributes and programs routes for pod traffic between nodes using its data store. {{site.prodname}} supports unencapsulated traffic within a single subnet, as well as selective VXLAN encapsulation for clusters that span multiple subnets.
```

Comment:
Better to say "Calico Routing..." rather than "Calico..." or some other qualifier since Calico doesn't always do this stuff.  Maybe say without using BGP too?